jax/_src/numpy/lax_numpy.py

```diff
@@ -4462,10 +4462,18 @@ def vander(x, N=None, increasing=False):
 
 ### Misc
 
+_ARGWHERE_DOC = """\
+Because the size of the output of ``argwhere`` is data-dependent, the function is not
+typically compatible with JIT. The JAX version adds the optional `size` argument which
+specifies the size of the leading dimension of the output; it must be specified statically
+for ``jnp.argwhere`` to be traced. If specified, the indices of the first `size` True elements
+will be returned; if there are fewer nonzero elements than `size` indicates, the index arrays
+will be zero-padded.
```

Comment:
Just an idea - "If specified" on its own here may require some users to reread the whole thing, so this may just ease the cognitive overload a little:

```suggestion
typically compatible with JIT. The JAX version adds the optional `size` argument, which
specifies the size of the leading dimension of the output - it must be specified statically
for ``jnp.argwhere`` to be traced. If `size` is specified, the indices of the first `size` True
elements will be returned; and if there are fewer nonzero elements than `size` indicates, the index
arrays will be zero-padded.
```

Is the max char limit 80 or 100 btw

Reply:
JAX conforms to no style guide – make the lines as long as you wish :grin:

Reply:
Done, thanks